packages/terra-application-links/src/tabs/_Tab.jsx

```diff
@@ -20,33 +16,50 @@ const propTypes = {
    * The display text for the tab.
    */
   text: PropTypes.string.isRequired,
+  /**
+   * The click callback of the tab.
+   */
+  onTabClick: PropTypes.func,
 };
 
 const ApplicationTab = ({
-  isCollapsed,
+  onTabClick,
   path,
   text,
   ...customProps
-}) => {
-  const tabClassNames = cx([
-    { tab: !isCollapsed },
-    { 'collapsed-tab': isCollapsed },
-    customProps.className,
-  ]);
-
-  const tabRole = {};
-  if (!isCollapsed) {
-    tabRole.role = 'tab';
-  }
+}) => (
+  <Route
+    render={({ location, history }) => {
+      const isActive = !!matchPath(location.pathname, { path });
+      const tabClassNames = cx([
+        'tab',
+        { 'is-disabled': isActive },
+        customProps.className,
+      ]);
+      const tabAttr = { 'aria-current': isActive, role: 'tab' };
 
-  return (
-    <NavLink {...customProps} {...tabRole} className={tabClassNames} to={path} key={path} activeClassName={cx(['selected'])}>
-      <span className={cx(['tab-inner'])}>
-        {text}
-      </span>
-    </NavLink>
-  );
-};
+      return (
+        <button
```

Comment:
Do we want to use terra-button to enable consistent keyboard accessibility between browsers? or are all focus & active styling removed for these buttons?

Reply:
Terra button no longer accepts children or truncates text, so we need to managed it ourselves, also the the styles require a lot of overrides, so we just went with a basic button.